packages/ui/src/theme/tokens/components/checkboxField.ts

```diff
@@ -1,4 +1,10 @@
-export const checkboxfield = {
+export interface CheckboxFieldTokens {
+  alignItems: any;
```

Comment:
I will follow up with more PR's to type all the other component files. For now, typing these as any is the same as what was there before.